### PR TITLE
Add review function to FAME

### PR DESCRIPTION
**Pull request by Tiagojdc**
### Changes
Add a review features which will help analyst to work in collaboration.

It adds a permission named "review" which allow users to access to a new page. This page contain sames elements as on index page, but with the possibility to review them and filter reviewed elements

### CODE REVIEW

File: web/templates/files/details.html
Comment (gaelmuller) [inline web/templates/files/details.html:22]: We can't do this because `analysis` will not be defined when looking at the object's page.

File: web/templates/files/details.html
Comment (gaelmuller) [inline web/templates/files/details.html:21]: This should only appear if the user has the appropriate permission

**Review by gaelmuller — CHANGES_REQUESTED** [review]
Thank you for your contribution ! I think this is a useful feature but it would need some adjustments before we can merge it in my opinion:

* It is currently introducing a bug that is breaking the "Object Details" page
* I think that the review status should be handled like the sharing groups: be attached to the object and duplicated inside analyses
* I don't see the point of having a different page. We could use the analyses page directly and use the permission to display (or not) the additional column & filters

**Comment by Augustin-FL** [comment]
Hi @gaelmuller 

May I request a second review of this PR ?

If needed, I have merged the PR to a temporary branch https://github.com/Augustin-FL/fame/tree/review, you can test it with docker:
```
git clone https://github.com/Augustin-FL/fame.git
cd fame/docker
git checkout review
cp fame.env.template fame.env
echo "DEFAULT_EMAIL=admin@changeme.fame" >> fame.env
echo "DEFAULT_PASSWORD=changeme" >> fame.env
docker-compose up --build
```

**Comment by Augustin-FL** [comment]
Hi,

I will merge this. 
if you have some comments to make on this PR, feel free to let them here (even after the merge)

**Pull request merged**
